lib/src/dynamic_color_builder.dart

```diff
@@ -57,15 +62,28 @@ class DynamicColorBuilderState extends State<DynamicColorBuilder> {
       debugPrint('Failed to obtain dynamic colors.');
     }
 
-    // If the widget was removed from the tree while the asynchronous platform
-    // message was in flight, we want to discard the reply rather than calling
-    // setState to update our non-existent appearance.
-    if (!mounted) return;
-
-    setState(() {
-      _light = corePalette?.toColorScheme();
-      _dark = corePalette?.toColorScheme(brightness: Brightness.dark);
-    });
+    if (corePalette != null) {
+      setState(() {
+        _light = corePalette?.toColorScheme();
```

Comment:
```suggestion
        _light = corePalette.toColorScheme();
```

Reply:
This will say `The method 'toColorScheme' can't be unconditionally invoked because the receiver can be 'null'.`.

<img width="799" alt="image" src="https://user-images.githubusercontent.com/833473/170062734-fd3967d4-c522-4164-b196-2f4d6a2acbcc.png">

Reply:
Now that you're not checking whether it's null first, yeah.